### PR TITLE
Update collection

### PR DESCRIPTION
**Pull request by evgeni**
see https://galaxy.ansible.com/docs/contributing/creating_collections.html and https://docs.ansible.com/ansible/devel/collections_tech_preview.html for `issues` key and the `tests` folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ contained in the repository for use, testing, or development of new modules.
 
 ## How to write modules in this repository
 
-First of all, please have a look at the [Ansible module development](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html) guide and get familiar with the general Ansible module layout.
+First of all, please have a look at the [Ansible module development](https://docs.ansible.com/ansible/latests/dev_guide/developing_modules_general.html) guide and get familiar with the general Ansible module layout.
```
Comment (mdellweg) [inline README.md]: Nope!

Comment (evgeni) [reply README.md]: DAMN

**Comment by evgeni** [comment]
:green_apple:

File: galaxy.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-namespace: "foreman"
+namespace: "theforeman"
 name: "foreman"
 authors:
   - "Andrew Kofink <ajkofink@gmail.com>"
```
Comment (mdellweg) [inline galaxy.yml]: BTW: This list is a little bit outdated. Does not need to be fixed here...

**Pull request merged**